docs/dev/table/functions/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Functions"
+nav-id: table_functions
+nav-parent_id: tableapi
+nav-pos: 116
```

Comment:
This renders very low down, I think functions should be higher up on the side nav.